### PR TITLE
filters by tag and id

### PR DESCRIPTION
**Pull request by jeanluciano**
# Description
Fixes # (issue)

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status
- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by robert-petersen — APPROVED** [review]
Looks good code-wise!

**Review by Brandonx1123 — APPROVED** [review]
The code looks really good and makes sense logically.

**Review by john-chamberlin — APPROVED** [review]
Look solid

File: api/incidents/incidentsRouter.js [hunk 1]
```diff
@@ -429,6 +429,9 @@ router.get('/download', async (req, res) => {
     const state = req.query.state || null;
     let start = req.query.start || null;
     let end = req.query.end || null;
+    let selectedIncidents = req.body.selectedIncidents || null;
+    let selectedTags = req.body.selecteTags || null;
+    console.log(req.body);
```
Comment (ryan-hamblin) [inline api/incidents/incidentsRouter.js:434]: please remove all `console.logs()`  this is actually a potential security risk as well. we shouldn't print the req.body in production. It's really good for debugging in development, but shouldn't be left in production.

File: api/incidents/incidentsRouter.js [hunk 2]
```diff
@@ -455,6 +458,23 @@ router.get('/download', async (req, res) => {
       incidents = filterDataByDate(incidents, range);
     }
```
Comment (ryan-hamblin) [inline api/incidents/incidentsRouter.js:460]: spacing here is different than lines 465 and 466. please pick one convention and go with that.

**Review by ryan-hamblin — COMMENTED** [review]
overall this work is very clean there are a few things I've requested that you change.

**Pull request merged**